test/public/dataPasses/overviewPerLhcPeriod.test.js

```diff
@@ -160,7 +155,6 @@ module.exports = () => {
         // Sort by name in an ascending manner
         const nameHeader = await page.$('th#name');
         await nameHeader.evaluate((button) => button.click());
-        await waitForTimeout(300);
```

Comment:
Here if you remove the waitForTimeout, you need to find a way to be sure the table has been ordered
You can have a look at logs/overview.test.js line 278 how I did it

Reply:
Done